homeassistant/components/samsungtv/bridge.py

```diff
@@ -141,8 +141,8 @@ def send_key(self, key):
         except (UnhandledResponse, AccessDenied):
             # We got a response so it's on.
             LOGGER.debug("Failed sending command %s", key, exc_info=True)
-        except OSError:
-            # Different reasons, e.g. hostname not resolveable
+        except (AttributeError, OSError):
```

Comment:
It would be better to not access the remote at all when we know it's None. We set it to None ourselves above. So we should adjust that whole flow instead to avoid the access.